server/src/controllers/eventController.service.ts

```diff
@@ -363,48 +361,80 @@ interface checkoutForm {
   seatingAcc: string;
   comments: string;
   optIn: boolean;
+  donorbadge?: boolean;
+  vip?: boolean;
+  volunteerlist?: boolean;
```

Comment:
It might be a good idea to renamed `checkoutForm` to something else since it's being used for multiple purposes besides just the checkoutForm. Probably would be a good idea to also switch from `optIn` to just `newsletter` at some point for consistency.